aesara/tensor/math.py

```diff
@@ -467,6 +467,9 @@ def infer_shape(self, fgraph, node, shapes):
         )
         return [rval]
 
+    def R_op(self, inputs, eval_points):
+        raise ValueError("Argmax is not a differentiable operation")
+
```

Comment:
```suggestion
```
There's already a `grad` implementation&mdash;albeit effectively non-functional&mdash;so I don't know if it helps to have an `R_op` like this.  Also, I don't think a `ValueError` would be the best here.

If a gradient is undefined, I believe we should return an `aesara.gradient.grad_undefined`-generated `NullType`.  Same with unimplemented gradients: we should use `aesara.gradient.grad_not_implemented`.